listenbrainz/webserver/static/js/src/stats/UserHistory.tsx

```diff
@@ -303,24 +315,24 @@ export default class UserHistory extends React.Component<
                   data-toggle="dropdown"
                   type="button"
                 >
-                  {entity}s
+                  {`${entity}s`}
                   <span className="caret" />
                 </button>
                 <ul className="dropdown-menu" role="menu">
                   <li>
                     <a
-                      href="#"
+                      href=""
```

Comment:
I know this is not a good idea but bootstrap supports only `anchor` tags in dropdown-menu and pager, a solution for this can be using `react-bootstrap`, which provides us with the components needed in a `React` friendly way.

Reply:
Does this work without an `href` attribute at all?
The [HTML5 specs](https://www.w3.org/TR/2016/REC-html51-20161101/textlevel-semantics.html#the-a-element) say it should be valid:
>If the a element has no href attribute, then the element represents a placeholder for where a link might otherwise have been placed, if it had been relevant, consisting of just the element’s contents.

Reply:
It does work but the cursor switches to an `I-Beam` and `ESlint` throws more errors.